modules/ROOT/pages/bat-example-test-suite.adoc

```diff
@@ -0,0 +1,146 @@
+  # - type: Email
+  #   options:
+  #     EMAILS: <email_list>
+  # - type: NewRelic
+  #   options:
+  #     NEW_RELIC_LICENSE_KEY: <license_key>
+  # - type: JSON
+  #   outFile: /tmp/JSON.json
+  # - type: JUnit
+  #   outFile: /tmp/JUnit.xml
+----
+
+Every test suite must contain a `bat.yaml` file at its root level.
+
+=== The `suite` Section of the example `bat.yaml` File
+The `suite` section names the suite. You can use any names that you like for the suites that you write.
+
+=== The `files` Section of the example `bat.yaml` File
+
+This example test suite contains only one test. The test is listed in the `files` section as the file `HelloWorld.dwl` in the `tests` folder.
+
+As you create tests, you add references to them in your `bat.yaml` file. For example, suppose that in your test-suite directory you create a folder named `api-tests` and write a test named `first-test.dwl` in that folder. In the `files` section of your `bat.yaml` file, you add the following indented line:
+
+----
+  file: api-tests/first-test.dwl
+----
+
+If you write a second test, this one named `second-test.dwl`, you add this line after that one:
+
+----
+  file: api-tests/second-test.dwl
+----
+
+The order in which you list the tests determines the order in which the BAT CLI runs them. In this example, the test `prodInfo.dwl` runs before the test `shoppingCart.dwl`:
+
+----
+files:
+  file: retail-site-tests/orderFullfillment.dwl
+  file: retail-site-tests/shoppingCart.dwl
+----
+
+To run the `orderFullfillment.dwl` test after the `shoppingCart.dwl` test, you can change the order in which they are listed:
+
+----
+files:
+  file: retail-site-tests/shoppingCart.dwl
+  file: retail-site-tests/orderFullfillment.dwl
+----
+
+Finally, you can create more than one folder to contain the tests in a test suite. You can even create subfolders within the folders that contain your tests. Just be sure that you provide the correct relative path when you list a test in the `files` section of your `bat.yaml` file.
+
+=== The `reporters` Section of the `bat.yaml` File
+
+This section is for configuring various tools or third-party applications to report test failures. In this example, a failure of the `HelloWorld.dwl` test prompts the BAT CLI to write a report of the failure to the file `/tmp/HTML.html`. For information about setting up reports, see xref:bat-reporting-task.adoc[Configure and View Test Reports].
+
+=== The `secrets` Section of the `bat.yaml` file
+
+If you plan to run a test suite from a private location, you can use the BAT CLI's integration with Anypoint Secrets Manager to use shared secrets in your tests.
+
+=== The `config` Folder
+
+You can store variables in the files in the `config` folder. For example, let's say you're developing an API and you deploy it to two different environments: `dev` and `qa`. The endpoint url for the first is `dev.myapi.com` and for the second `qa.myapi.com`. The API should behave the same in both environments.
```

Comment:
replacing "let's say" to ease translation
```suggestion
You can store variables in the files in the `config` folder. For example, imagine you're developing an API and you deploy it to two different environments: `dev` and `qa`. The endpoint URL for the first is `dev.myapi.com` and for the second `qa.myapi.com`. The API should behave the same in both environments.
```

Reply:
Is there a disconnect in saying that the API should behave the same in both environments, followed by saying that you want to test it in each?
Also, is "should" a direction or a prediction?

Reply:
GLOBAL: I recommend spelling out "configuration," when not referring to a syntax value or variable.
I haven't marked it, but "config" appears twice in the following text, and most likely elsewhere.
Suggest search and replace if you agree.

Reply:
Done.